### ISSUE TITLE
lpush and rpush should support multiple values

### ISSUE THREAD

**Issue by nfg** [jlavallee/Test-Mock-Redis#25]
Hey! I stumbled across a TODO test at $WORK where LRANGE was returning unexpected data. It turns out that the call to [LPUSH](https://redis.io/commands/lpush) sent multiple values, but the implementation only accepts a single value.

Here's what I get when I test via the command line client:

```
127.0.0.1:6379> LPUSH yarr foo bar
(integer) 2
127.0.0.1:6379> LRANGE yarr 0 4
1) "bar"
2) "foo"
```

Ditto for [RPUSH](https://redis.io/commands/rpush). [LPUSHX](https://redis.io/commands/lpushx) and [RPUSHX](https://redis.io/commands/rpushx) only accept single values

**Comment by valcomm** [comment]
I think this issue might be closed as 0.22 supports multiple values for lpush and rpush

**Comment by jlavallee** [comment]
Thanks!
